lib/combobox.js

```diff
@@ -238,18 +238,6 @@ module.exports = React.createClass({
     this.refs.input.select();
   },
 
-  inputKeydownMap: {
-    8: 'removeLastToken',
-    9: 'selectOnEnter',
-    13: 'selectOnEnter',
-    27: 'hideOnEscape',
-    32: 'selectOnEnter',
-    38: 'focusPrevious',
-    40: 'focusNext',
-    186: 'selectOnEnter',
-    188: 'selectOnEnter'
-  },
```

Comment:
@dbslone @garronmichael This makes sense to me.  How would we remove a default key mapping using this strategy? The reason I did it like this is because for the new onboarding slide we need to remove the selectOnEnter